packages/core/content-releases/server/src/controllers/release-action.ts

```diff
@@ -24,6 +25,17 @@ type LocaleDictionary = {
   [key: Locale['code']]: Pick<Locale, 'name' | 'code'>;
 };
 
+const getGroupName = (queryValue?: 'action' | 'locale') => {
+  switch (queryValue) {
+    case 'action':
+      return 'type';
+    case 'locale':
+      return 'entry.locale.name';
+    default:
+      return 'entry.contentType.displayName';
```

Comment:
I'm surprised that the default value is not one that you could explicitly request via the queryValue

Reply:
Yeah originally I had it then I removed it right before my commit. It seemed weird to have it as both default and an option but I can add it back if you prefer.

Reply:
yeah i think having it makes the frontend easier to build